diff --git a/core/modules/media_library/src/Plugin/views/field/MediaLibrarySelectForm.php b/core/modules/media_library/src/Plugin/views/field/MediaLibrarySelectForm.php
index 2d0702b..c643194 100644
--- a/core/modules/media_library/src/Plugin/views/field/MediaLibrarySelectForm.php
+++ b/core/modules/media_library/src/Plugin/views/field/MediaLibrarySelectForm.php
@@ -130,7 +130,9 @@ class MediaLibrarySelectForm extends FieldPluginBase {
     return \Drupal::service('media_library.opener_resolver')
       ->get($state)
       ->getSelectionResponse($state, $selected_ids)
-      ->addCommand(new CloseDialogCommand());
+      ->addCommand($state->getOpenerId() === 'media_library.opener.cohesion'
+        ? new CloseDialogCommand('#modal-body')
+        : new CloseDialogCommand());
   }
 
   /**
