diff --git a/src/AcquiaDrupalEnvironmentDetector.php b/src/AcquiaDrupalEnvironmentDetector.php
index 1417858..7b6e92d 100644
--- a/src/AcquiaDrupalEnvironmentDetector.php
+++ b/src/AcquiaDrupalEnvironmentDetector.php
@@ -2,12 +2,16 @@
 
 namespace Acquia\DrupalEnvironmentDetector;
 
-class AcquiaDrupalEnvironmentDetector {
+use Symfony\Component\Console\Input\ArgvInput;
+
+class AcquiaDrupalEnvironmentDetector
+{
   /**
    * Is AH env.
    */
-  public static function isAhEnv() {
-    return (bool) self::getAhEnv();
+  public static function isAhEnv()
+  {
+    return (bool)self::getAhEnv();
   }
 
   /**
@@ -25,7 +29,8 @@ class AcquiaDrupalEnvironmentDetector {
    *
    * @see https://git.drupalcode.org/project/acsf/blob/8.x-2.62/acsf_init/lib/sites/default/acsf.settings.php#L14
    */
-  public static function isAcsfEnv($ah_group = NULL, $ah_env = NULL) {
+  public static function isAcsfEnv($ah_group = NULL, $ah_env = NULL)
+  {
     if (is_null($ah_group)) {
       $ah_group = self::getAhGroup();
     }
@@ -44,7 +49,8 @@ class AcquiaDrupalEnvironmentDetector {
   /**
    * Is AH prod.
    */
-  public static function isAhProdEnv() {
+  public static function isAhProdEnv()
+  {
     $ah_env = self::getAhEnv();
     // ACE prod is 'prod'; ACSF can be '01live', '02live', ...
     return $ah_env == 'prod' || preg_match('/^\d*live$/', $ah_env);
@@ -53,7 +59,8 @@ class AcquiaDrupalEnvironmentDetector {
   /**
    * Is AH stage.
    */
-  public static function isAhStageEnv() {
+  public static function isAhStageEnv()
+  {
     $ah_env = self::getAhEnv();
     // ACE staging is 'test' or 'stg'; ACSF is '01test', '02test', ...
     return preg_match('/^\d*test$/', $ah_env) || $ah_env == 'stg';
@@ -62,7 +69,8 @@ class AcquiaDrupalEnvironmentDetector {
   /**
    * Is AH dev.
    */
-  public static function isAhDevEnv() {
+  public static function isAhDevEnv()
+  {
     // ACE dev is 'dev', 'dev1', ...; ACSF dev is '01dev', '02dev', ...
     return (preg_match('/^\d*dev\d*$/', self::getAhEnv()));
   }
@@ -70,7 +78,8 @@ class AcquiaDrupalEnvironmentDetector {
   /**
    * Is AH ODE.
    */
-  public static function isAhOdeEnv($ah_env = NULL) {
+  public static function isAhOdeEnv($ah_env = NULL)
+  {
     if (is_null($ah_env)) {
       $ah_env = self::getAhEnv();
     }
@@ -81,52 +90,59 @@ class AcquiaDrupalEnvironmentDetector {
   /**
    * Is AH IDE.
    */
-  public static function isAhIdeEnv() {
+  public static function isAhIdeEnv()
+  {
     return strtolower(self::getAhEnv()) == 'ide';
   }
 
   /**
    * Is AH devcloud.
    */
-  public static function isAhDevCloud() {
+  public static function isAhDevCloud()
+  {
     return (!empty($_SERVER['HTTP_HOST']) && strstr($_SERVER['HTTP_HOST'], 'devcloud'));
   }
 
   /**
    * Get AH group.
    */
-  public static function getAhGroup() {
+  public static function getAhGroup()
+  {
     return getenv('AH_SITE_GROUP');
   }
 
   /**
    * Get AH env.
    */
-  public static function getAhEnv() {
+  public static function getAhEnv()
+  {
     return getenv('AH_SITE_ENVIRONMENT');
   }
 
   /**
    * Get AH realm.
    */
-  public static function getAhRealm() {
+  public static function getAhRealm()
+  {
     return getenv('AH_REALM');
   }
 
   /**
    * Get AH non production.
    */
-  public static function getAhNonProduction() {
+  public static function getAhNonProduction()
+  {
     return getenv('AH_NON_PRODUCTION');
   }
-    
+
   /**
    * Get AH application UUID.
    */
-  public static function getAhApplicationUuid() {
+  public static function getAhApplicationUuid()
+  {
     return getenv('AH_APPLICATION_UUID');
   }
-  
+
   /**
    * The path to the persistent file storage mount.
    *
@@ -135,7 +151,8 @@ class AcquiaDrupalEnvironmentDetector {
    *
    * @see https://docs.acquia.com/acquia-cloud/manage/files/about/
    */
-  public static function getAhFilesRoot() {
+  public static function getAhFilesRoot()
+  {
     return '/mnt/files/' . self::getAhGroup() . '.' . self::getAhEnv();
   }
 
@@ -145,14 +162,39 @@ class AcquiaDrupalEnvironmentDetector {
    * @return string|null
    *   ACSF db name.
    */
-  public static function getAcsfDbName() {
+  public static function getAcsfDbName()
+  {
     return isset($GLOBALS['gardens_site_settings']) && self::isAcsfEnv() ? $GLOBALS['gardens_site_settings']['conf']['acsf_db_name'] : NULL;
   }
 
+  /**
+   * Get a standardized site / db name.
+   *
+   * On ACE or simple multisite installs, this is the site directory under
+   * 'docroot/sites'.
+   *
+   * On ACSF, this is the ACSF db name.
+   *
+   * @param string $site_path
+   *   Directory site path.
+   *
+   * @return string|null
+   *   Site name.
+   */
+  public static function getSiteName($site_path)
+  {
+    if (self::isAcsfEnv()) {
+      return self::getAcsfDbName();
+    }
+
+    return str_replace('sites/', '', $site_path);
+  }
+
   /**
    * If this isn't a Cloud environment, assume it's local.
    */
-  public static function isLocalEnv() {
+  public static function isLocalEnv()
+  {
     return !self::isAhEnv();
   }
 }
