diff --git a/src/Entity/NextEntityTypeConfig.php b/src/Entity/NextEntityTypeConfig.php
index 7f6cf4e..602c061 100644
--- a/src/Entity/NextEntityTypeConfig.php
+++ b/src/Entity/NextEntityTypeConfig.php
@@ -23,6 +23,7 @@ use Drupal\next\SiteResolverPluginCollection;
  *   ),
  *   handlers = {
  *     "list_builder" = "Drupal\next\NextEntityTypeConfigListBuilder",
+ *     "views_data" = "Drupal\next\NextEntityTypeConfigViewsData",
  *     "form" = {
  *       "add" = "Drupal\next\Form\NextEntityTypeConfigForm",
  *       "edit" = "Drupal\next\Form\NextEntityTypeConfigForm",
diff --git a/src/Entity/NextEntityTypeConfigViewsData.php b/src/Entity/NextEntityTypeConfigViewsData.php
new file mode 100644
index 0000000..96ecb62
--- /dev/null
+++ b/src/Entity/NextEntityTypeConfigViewsData.php
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\next\Entity;
+
+use Drupal\views\EntityViewsData;
+
+/**
+ * Provides Views data for Alert entities.
+ */
+class NextEntityTypeConfigViewsData extends EntityViewsData {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getViewsData(): array {
+    $data = parent::getViewsData();
+
+    // Additional information for Views integration, such as table joins, can be
+    // put here.
+
+    return $data;
+  }
+
+}
diff --git a/src/Entity/NextSite.php b/src/Entity/NextSite.php
index 7199bbe..e18584b 100644
--- a/src/Entity/NextSite.php
+++ b/src/Entity/NextSite.php
@@ -25,6 +25,7 @@ use Drupal\jsonapi\ResourceType\ResourceType;
  *   ),
  *   handlers = {
  *     "list_builder" = "Drupal\next\NextSiteListBuilder",
+ *     "views_data" = "Drupal\next\NextSiteViewsData",
  *     "form" = {
  *       "add" = "Drupal\next\Form\NextSiteForm",
  *       "edit" = "Drupal\next\Form\NextSiteForm",
diff --git a/src/Entity/NextSiteViewsData.php b/src/Entity/NextSiteViewsData.php
new file mode 100644
index 0000000..fc92b36
--- /dev/null
+++ b/src/Entity/NextSiteViewsData.php
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\next\Entity;
+
+use Drupal\views\EntityViewsData;
+
+/**
+ * Provides Views data for Alert entities.
+ */
+class NextSiteViewsData extends EntityViewsData {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getViewsData(): array {
+    $data = parent::getViewsData();
+
+    // Additional information for Views integration, such as table joins, can be
+    // put here.
+
+    return $data;
+  }
+
+}
